pbe/breakupCoalescenceTables.py

```diff
@@ -0,0 +1,51 @@
+import fluid
+import numpy as np
+import matplotlib.pyplot as plt
+
+path = 'validationData/tables/1/'
+f = open(path + 'parameters', 'w')
+f_br = open(path + 'breakup', 'w')
+f_c = open(path + 'coalescence', 'w')
+
+F = fluid.fluid('coulaloglou', 0, 0)
+F.C1 = 1.52e-02
+F.C2 = 6.78e-02
+F.C3 = 1.06e-12
+F.C4 = 5.13e13
```

Comment:
I still don't see a benefit of this:
for i in arange(2):
    f.write('C' + repr(i+1) + ' ' + repr(F.Cbr[i]) + '\n')
for i in arange(2):
    f.write('C' + repr(i+3) + ' ' + repr(F.Ccoal[i]) + '\n')
over this:
f.write('C1 ' + repr(F.C1) + '\n')
+f.write('C2 ' + repr(F.C2) + '\n')
+f.write('C3 ' + repr(F.C3) + '\n')
+f.write('C4 ' + repr(F.C4) + '\n')

On the contrary - the current one seems to me more clear and concise.